### ISSUE TITLE
Draft blog post for new home page design

### ISSUE THREAD

**Issue by jennmalcolm** [ONRR/doi-extractives-data#3369]

**Comment by brentryanjohnson** [comment]
I think this should be two posts:

- How we decided to rebuild the homepage (@brentryanjohnson and @mcharg)
- The rebuild development process (@brentryanjohnson and @master12)

With editorial approval from @jennmalcolm.

**Comment by jennmalcolm** [comment]
I agree @brentryanjohnson. I think it makes sense to break this into two. Let's start with How we decided to rebuild the homepage. That's what this issue should be about. Then we can create another one for the rebuild process.
